packages/cli/package.json

```diff
@@ -52,7 +52,7 @@
     "@svgr/plugin-svgo": "6.3.x",
     "@svgr/rollup": "6.3.x",
     "@svgr/webpack": "6.3.x",
-    "@swc/core": "^1.2.239",
+    "@swc/core": "^1.3.9",
```

Comment:
Just one final thing as a result of this.

- You need to make a separate changeset for the cli as well, now. Patch level is fine.
- Can you change the same thing in `storybook/package.json` too? That doesn't need a changeset.
- After this change, users will need to be on a sufficiently recent CLI to be able to build. I wonder if there will be any complex effects of that. I think it would be good if you could make a completely separate changeset for `@backstage/create-app`, at a patch level, explicitly pointing out that we have updated to a bug fixed version of SWC and that you may want to run `yarn backstage-cli versions:bump` to get on the latest version including the CLI itself. The changeset for that package is looked at extra carefully by a lot of people.